packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -10,12 +10,15 @@
         background-attachment: fixed;
         background-size: cover;
         content: '';
+        height: 100%;
+        left: calc(-1*var(--terra-consumer-nav-width, 320px));
```

Comment:
use a transform to do this.. its more performant in the browser.  We may also want to use a 3d transform if thats still needed to trigger hardware acceleration.

https://developer.mozilla.org/en-US/docs/Web/CSS/transform?v=b

Reply:
Here is the reference on performance https://css-tricks.com/tale-of-animation-performance/